src/borg/testsuite/archiver.py

```diff
@@ -262,7 +262,7 @@ def cmd(self, *args, **kw):
         return output
 
     def create_src_archive(self, name):
-        self.cmd('create', '--compression=none', self.repository_location + '::' + name, src_dir)
+        self.cmd('create', '--compression=lz4', self.repository_location + '::' + name, src_dir)
```

Comment:
lz4 is the default. I did the change so I can comment on it:

when I put --compression=zlib here, another strangeness appears: the test intended to corrupt the chunk by appending b'1234' is not effective. likely because the zlib decompressor does not care for appended data and just returns the expected, correct data.